### ISSUE TITLE
During google developers setup, can't find option for "other"

### ISSUE THREAD

**Issue by robohyun66** [kidd/org-gcal.el#95]
Once you’ve set up a consent screen (this is required), click on Create Credentials and Oauth client ID with Application type **Other.**

I can't find the option **Other**. Are these instructions outdated?

**Comment by rnfein** [comment]
Thought **Other** is missing, you should be able to configure this as a **Desktop Client**.

**Comment by telotortium** [comment]
Yeah, Google changes around the names of these options every so often. @robohyun66 has the suggestion from @rnfein worked for you?

**Comment by reedspool** [comment]
**Desktop Client** worked fine for me
